.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug Report
+about: Report a bug encountered while using nitric
+labels: bug
+---
+
+**Describe the bug**
+<!--
+A clear and concise description of what the bug is. What happened?
+-->
+
+**Expected behavior**
```

Comment:
Are we appealing to american english with behavior vs behaviour? Or should this be changed to everywhere else language?

Reply:
This is as written by @elisselockhart - I suspect the answer is yes..

Reply:
This is the spelling that was there previously, but I don't think we've made a conscious decision on which spellings to use as a company.